app/src/main/java/me/capcom/smsgateway/modules/localserver/routes/MessagesRoutes.kt

```
@@ -217,6 +218,34 @@ class MessagesRoutes(
                 )
             )
         }
+        delete("{id}") {
+            if (!requireScope(AuthScopes.MessagesCancel)) return@delete
+            val id = call.parameters["id"]
+                ?: return@delete call.respond(HttpStatusCode.BadRequest)
+
+            val message = try {
+                messagesService.cancelMessage(id)
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)